setup.py

```diff
@@ -39,15 +39,50 @@
         'importlib-resources>=5.1.4;python_version<"3.7"',
     ],
     extras_require={
-        "ner": ["torch>=1.1.0", "torchtext==0.4.0", "tqdm>=4.36.1"],
-        "test": ["ddt", "testfixtures"],
-        "example": ["termcolor==1.1.0", "hypothesis==2.0.0"],
+        "models": [
+            "torch>=1.1.0",
+            "torchtext==0.4.0",
+            "tqdm>=4.36.1",
+            "texar-pytorch>=0.1.1",
+        ],
+        "test": [
+            "ddt",
+            "testfixtures",
+            "testbook",
+            "termcolor",
+            "transformers>=4.15.0",
+            "nltk",
+        ],
+        "example": [
+            "termcolor==1.1.0",
+            "hypothesis==2.0.0",
+            "texar-pytorch>=0.1.1",
+        ],
         "wikipedia": ["rdflib==4.2.2"],
+        "data_aug": [
+            "transformers>=3.1",
+            "nltk",
+            "texar-pytorch>=0.1.1",
+        ],
+        "subword_tokenize": ["texar-pytorch>=0.1.1"],
+        "texar-encoder": ["texar-pytorch>=0.1.1"],
         # transformers 4.10.0 will break the translation model we used here
-        "augment": ["transformers>=3.1, <=4.9.2", "nltk"],
         "stave": ["stave>=0.0.1.dev12"],
         "audio_ext": ["soundfile>=0.10.3"],
-        "remote": ["fastapi>=0.65.2", "uvicorn>=0.14.0",],
+        "remote": [
+            "fastapi>=0.65.2",
+            "uvicorn>=0.14.0",
+        ],
+        "extractor": ["texar-pytorch>=0.1.1"],
+        "rl-aug": ["texar-pytorch>=0.1.1"],
+        "uda": [
+            "texar-pytorch>=0.1.1",
+        ],
+        "mt": [
+            "transformers>=3.1",
+        ],
+        "dict": ["nltk"],
+        "emb_sim_rep_op": ["texar-pytorch>=0.1.1"],
```

Comment:
So, `rl-aug`, `uda`, `mt`, `dict`, `emb_sim_rep_op` are all under data aug modules, since we have a section for `data-aug` that have similar dependencies, maybe we just merge them. Actually having less options here would be better.